app/gulp/sass/app.scss

```diff
@@ -3,3 +3,25 @@
 @import '_error';
 @import '_layout';
 @import '_misc';
+
+html {
+  height: 100%;
+  width: 100%;
+}
+
+body {
+  height: calc(100% - 60px);
+  width: 100%;
+}
```

Comment:
This CSS is adding a tiny scrollbar to every page for some reason. Also, maybe we should avoid using `calc` if it's possible (I checked caniuse and 15% of people are using browsers that don't have it). I would also add layout-related CSS that goes on every page into `_layout.scss`.